eng/icu.proj

```diff
@@ -37,4 +39,18 @@
             Lines="@(_ItemsFromEmscriptenVersionFile->Replace('&quot;', ''))"
             Overwrite="true" />
   </Target>
+  <Target Name="PatchWasiSdk" Condition="'$(TargetOS)' == 'Wasi'">
```

Comment:
Please rename to `DownloadWasiSdk`. Also I'm not clear if this is necessary when running on docker image with wasi SDK already in place.

Reply:
Updated, now it will download wasi sdk only when needed.